src/pages/en/guides/integrations-guide.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Using Integrations
+---
+
+**Astro Integrations** can extend Astro with new functionality and behaviors. Astro's Integration API is inspired by Rollup and builds on top of Vite's already impressive plugin API. Taken together, this system gives you access to configure just about everything in Astro, from high-level Astro dev/build features to low-level file loading & build logic. 
+
+## Adding an Integration
+
+To use a integration, it first needs to be available in your project. This usually means installing it from npm using your favorite package manager of choice:
+
+```
+npm install --save-dev @astrojs/react
+```
+
+Then, you will need to add the integration to your `astro.config.mjs` project configuration file.
+
+```diff
+  import { defineConfig } from 'astro/config';
++ import react from '@astrojs/react';
+
+  // https://astro.build/config
+  export default defineConfig({
++   integrations: [react()],
+  });
+``` 
+
```

Comment:
So, two steps? An import (at the top of the file, along with any other imports), and then updating the integrations array in defineConfig (the way previously we would have updated the renderers array in the old file)?

I might call attention explicitly to two steps, especially since this was only one step before.

"Then you will need to *import and add* the integration...."  ??